scripts/operations/configuration/python_lib/product_catalog.py

```diff
@@ -0,0 +1,210 @@
+#
```

Comment:
There isn't anything inherently wrong with this, but I want to point out that we actually already publish a python client library for the product catalog: https://artifactory.algol60.net/ui/native/csm-python-modules/simple/cray-product-catalog/

Some of the things it does are being duplicated here. Some of the things here are things that the library above does not do. It probably would be more work to correctly set up a dependency on that library.

Reply:
Short term it would be more work, probably, but it's good to know about that. Long term it probably makes sense to not reinvent the wheel here, if possible. Especially since the underlying structure of CPC is supposed to eventually move away from being based on a K8s config map -- I presume if/when that happens, the CPC Python client library will be updated accordingly, so using it would reduce the changes needed here at that time.

Reply:
Captured this in [CASM-3740](https://jira-pro.its.hpecorp.net:8443/browse/CASM-3740)